indico/modules/events/surveys/models/surveys.py

```diff
@@ -98,6 +101,36 @@ class Survey(db.Model):
         nullable=False,
         default=False
     )
+    #: Whether start notification has been already sent
+    start_notification_sent = db.Column(
+        db.Boolean,
+        nullable=False,
+        default=False
+    )
+    #: Whether to send survey related notifications to users
+    notifications_enabled = db.Column(
+        db.Boolean,
+        nullable=False,
+        default=False
+    )
+    #: Whether include Participants / Registrants when sending start notifications
+    include_participants_registrants = db.Column(
+        db.Boolean,
+        nullable=False,
+        default=False
+    )
+    #: List of email addresses that will get a notification about the start of a survey
+    start_notification_emails = db.Column(
+        JSON,
```

Comment:
it's always a list, so ARRAY makes more sense. `ARRAY(db.String)` and `default=[]`